components/engine/src/util.rs

```diff
@@ -59,9 +59,7 @@ pub fn delete_all_in_range_cf(
 ) -> Result<()> {
     let handle = rocks::util::get_cf_handle(db, cf)?;
     let wb = WriteBatch::new();
-    // Since CF_RAFT and CF_LOCK is usually small, so using
-    // traditional way to cleanup.
-    if use_delete_range && cf != CF_RAFT && cf != CF_LOCK {
+    if use_delete_range && cf != CF_LOCK {
```

Comment:
so should we add `assert!(Cf != CF_RAFT)`?

Reply:
I don't think so, it's totally ok to delete range on the raft cf, because it deletes nothing. After #4372 is merged, it returns an error because there is no such cf.